jitsi-media-transform/src/main/kotlin/org/jitsi/nlj/stats/EndpointConnectionStats.kt

```diff
@@ -144,20 +144,30 @@ class EndpointConnectionStats(
             // The delaySinceLastSr value is given in 1/65536ths of a second, so divide it by .000065536 to get it
             // in nanoseconds
             val remoteProcessingDelay = Duration.ofNanos((reportBlock.delaySinceLastSr / .000065536).toLong())
-            rtt = (Duration.between(srSentTime, receivedTime) - remoteProcessingDelay).toDoubleMillis()
-            if (rtt > 7.secs.toMillis()) {
+            if (remoteProcessingDelay > Duration.ofMinutes(5)) {
+                logger.warn("Ignoring report block with suspiciously long DLSR: $remoteProcessingDelay")
+                return@let
+            }
+
+            val newRtt = (Duration.between(srSentTime, receivedTime) - remoteProcessingDelay).toDoubleMillis()
+            if (newRtt > 7.secs.toMillis()) {
                 logger.warn(
-                    "Suspiciously high rtt value: $rtt ms, remote processing delay was " +
+                    "Ignoring suspiciously high rtt value: $newRtt ms, remote processing delay was " +
```

Comment:
7 seconds RTTs are actually possible, usually when someone is in a cellular network that's just marginally connected.  Maybe we should keep a stat for this separate from the RTTs?  (Not sure how best to handle repeated high RTTs from the same endpoint though.)

Reply:
What's a better cutoff?

Reply:
I think it's as reasonable a cutoff as any, it's certainly indicative of a very unhappy endpoint (or a very unhappy cloud instance, sometimes) -- I just think it'd be good to have the stat, too.  Doesn't need to be this PR though.